nss-day-cohort-41/react-nutshell-thepecans
# Refactor date object in events

# Description
Previously, attempts to edit a date crashed the app. I completely refactored the date format in the code and in the database. For everything to work, you MUST change your event dates in the database to this format: "2020-07-20T05:00:00.000Z". Everything from the "T" on should be written exactly as in that example; otherwise, you will encounter issues with the search and with the displayed date on the Event List page.
New and edited events conform to the new date format.

Fixes # 5
## Type of change
Please delete options that are not relevant.

# Testing Instructions
Navigate to "...events." Create and edit events. Make sure new and edited events do not generate any errors and are displayed with the correct date, in the correct order.

# Checklist:
